website/docs/cloud-docs/api-docs/notification-configurations.mdx

```diff
@@ -147,7 +149,13 @@ You can trigger a test notification from the workspace notifcation settings page
 
 ### Workspace Notification Payload
 
-Workspace events include detailed information about workspace-level validation events like [health assessments](/terraform/cloud-docs/workspaces/health) if you enable them for the workspace. Much of the information provides details about the associated [assessment result](/terraform/cloud-docs/api-docs/assessment-results), which Terraform Cloud uses to track instances of continuous validation. Generic notifications for workspace events contain the following information:
+Workspace events include detailed information about workspace-level validation events like [health assessments](/terraform/cloud-docs/workspaces/health) if you enable them for the workspace. Much of the information provides details about the associated [assessment result](/terraform/cloud-docs/api-docs/assessment-results), which Terraform Cloud uses to track instances of continuous validation.
+
+The types of attributes returned in the payload details will vary depending on the type of `trigger_scope`.
```

Comment:
This feels kinda vague? Do folks need to know what `trigger_scope` is or how it affects payload? We could link somewhere or expand here?
```suggestion
Terraform Cloud returns different types of attributes in the payload details, depending on the type of `trigger_scope`.
```